client/less/classic-modal.less

```diff
@@ -0,0 +1,45 @@
+.success {
```

Comment:
I need `.close` to be nested inside `.modal-header` to override some default modal styles